frontend/src/routes/team/+page.svelte

```diff
@@ -0,0 +1,35 @@
J6M3IufZRz+eJGHaXuulFezNHpg9uqqqtexVBZmRirPc/nk2D6j5CBH2z9E2svSo2Kr3ORxv04nJJLY8yT+pnW2io7uutM54VVcnz4QBn9pCG5fZLqms0mo1LV10graa/H3wKniVSMcI5gE8+XpJ0ECsREBPLUadLFKuqsp6hgCD9DPWIGj6vs6qrc3bOus0Nx5kV86GP8APQeXl5Ymub8azbS6O/TajR16jjHCNRpwTgE4Jag5KsR5jkJLcQMHuXsMaDRafTjqi+L3dvE5/UzOREBERAREQEREBERAREQEREC22loU1FVtNgzXYpVh7EYnK+so1Oydayhimo09nhfHxD7r46FWU9PcidYzm/tqfO1bfaqofsx/7wJn7Pt7k2pphZyW9PDag+63qP5W6j/abTOXdwd5W2TrFsdH7thwXJghuBsMHCkcyueIDzBPrmdNaDW131pbUwetwGVh0IMDBb5bl6bair3wZbUz3dqEhkzzPLoRkdCPLylnuFuc+gN9mouOo1DkItjcWRQgAROf1J+k3GICIiAiIgIiICIiAiIgIiICIiAiIgIiICIiAiIgJzT2xAvtXVKvXFaj5lBj/WdLTmzVn+0d4GxzV9YFH5aSFz8sVE/WBu/avuR3mkq1dK/b0VItoA5vUqjmR5lOvrjPoJZdg28xBt0FjcjmynPl/wARB7cww+bSZyARg9Jzbvls9th7VFlAwgYX1Dy4CfFX8viX2BEDpSJbbN1qaiqq6s5rsRXU+qsAw/1lzAREQEREBERAREQEREBERAREQEREBERAREQET5ZsczyEizfjteroL06ELbaMg2nnUp/l/wCIR7cveBI22tq1aWp7bbFRQpILEDJA5Aepz5SDewfZbXa99Qw8NNRJP97acD68Ic/UTD7L2FtTb1wsdndc877cipB/dryBPP4UHzxJ83P3Xp2ZQKacnJ4nc44rHwAWP6YA8hAzkjjtw2Cl+iGp4gtumOQT99HIDV/MkAj3HvJHM5+7Zt7TrNQNJSS1FLYIXJ72/pjHnwnwgepPpAkDsP2gbdmhG/8AxsZB+U+ID6cWJIU1ns73e/s/Q0UsALSOO3/Ebmwz5gfD9Js0BERAREQEREBERAREQEREBERAREQEREBMDvnvPVszTm+xWbnwoq/ecg4BP3Ry6zPTE70bDr1+mt09nRxyOOaMOauPcHnAgHentD1e1SlLMum07EBlVm4Tk4LW2YBK+wGOXnJL3P7JNHpgtmoI1NuARkYpX04a/vfNifpIJ2rs23S22UXrw2oSrDyPup81I5g+hkk7ndr7aWmmjU0NYtahBYjDj4VAA4lbkxAHXPOQTnVWFACgBR0AAAHyE+5g92d7dHtFeLTWhmHxIcrYn5kPP69PeZwyjV+0jeH+z9BfYpxcw7ur/EbkG/pGW+kjLsT3O7+w6+8ZqrOKQfv2DPFYfZeg9ST6TEdsu8Z1mtNK5FOmyg5/FZnxv9OSjz5H1mzbm9r2mopqo1GnaoIAitSA1fCB1K54gfYA9YEzRNd2Hvxs7WkLRqULn7jZR/8AI4BP0mw5gViIgIiICIiAiIgIiICIiAiIgIiICIiAiIgaT2hdntW1Atit3WqUYD4yrr+GxfPHkRzEi5+xvaYbAOnK/i7xh+3DOh4gR1uF2X1aBqtRc7PrELc0ZhXhuQBX72B9PblJFiIEfb2dk+j1rvdWW097ElinOt29WrPQnzK4/WRjt3sn2lpz9mg1KetRUEfNGIP6ZnR8piBztux2U67UW1/xNRo04OXYsneY/u1HFhs+Zk+bF2aNLRVQHdxWoXisbidsebN5mX2IgIiICIiAiIgIiICIiAiIgIiICIiAiIgIiICIiAiIgIiICIiAiIgIiICIiAiIgIiICIiAiIgIiICIlC0CsS30usWw2BfuOUbPqACce3iEpqdfXVnjcLhGc/kTHE30yIFzE+FsBwQff6T4v1KVqzswCKpZj6KBkmB7RPkOD0+c8U1iGxqx8SqrH0wxYDB9fCYFxE+S0cY9RA+olAZb2a+pbUpLjvWBZU+8VHVseQ9zAuYmMO39Nwd4LAycbJlA7ZZclgoUEnABJI5AAmezbVoDKveLxNWbRz5GtcZfPQL4hz94F7Exo25QVpZWZ1tGa+Cu1+IchxYVSQvMeI4HOfJ2/p8uAzMVPMJXc5Pi4SUCqS6huRZcgHkSIGUiYvT7w6aw1hXOX5LlLV58TKA3Eo4CWRgA2Cccsy+02qSzi4DkKxUnnjiHUA9Djpy8wR5QPaIiAiIgIiICIiAiIgJit49ntqKeBVRm4gRxkhQR0J8LcWOvCRg+3WZWfMDVNZuq7mxgahY7WEtgjiDVoFVuXTjrBxzx7zz1O6tl3emxdPx2pqVJ8Td33wTgKEpluEqfw/Fn2m4QIGoWbrWMztw1KWrKjhssAqPdGvu1QIAyZJOTjr8JPOeut3W4++WtKErfTNV0JPEVwvg4fAobxZB5+mec2mVEDT9VutbZx47mri5h0L8aDuhX3CjhXNWfFnI6/CDzlzTu/YLq7gtNfBwDuULGo4NmT8A8Q4wynh5EY88zZ4EDW9p7Ctue5sVA21BeMly9LBWBSvwjiRieZyp69cjFmd0mdizLSoIfhrXiKVFnpOKzwjkRU2Tgc36TbzAgYvZ2yBXUamOFF72IEZlCqbTYicsch0K9Oo6Suu09zX6d0So1pnjZnYPzBXCqKyDgHIyw6np1mUiBrT6DW93ai16deO0scXW8qyuCqt3HhbkBnB5E+0+9NsCyu1bQ+fA+aSU7sFkRVqWzuuLul4MZIz545kTYZWBq/wDY2oaulStKWIWHeLdcWrDOr5Qd2vGTjBVsDkOZ6T602wtRXgK9Y7umymlgW4ittiMXsHDyZRWAACcnJyM8tliBgNVsu7jpStav4WtVC/aOtisAV7wr3ZDlRzVeIc+ZPTHruvsizShw5UKQiqivY6goCGs4mAIZ8glRyHD1OSZm5WAiIgIiICIiB//Z"
```

Comment:
Bad practice. Upload the image to an image upload service like Cloudinary.

Reply:
This was already addressed and PR was already merged to main, so might not be needed